ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.tsx

```diff
@@ -635,27 +641,27 @@ export class SolarTimeline extends React.PureComponent<
     return (
       <div className={"solar-timeline-wrapper"}>
         <div className="header">
-          <PlayButton
-            tooltip={this._playLabel}
-            className="play-button"
-            isPlaying={this.state.isPlaying}
+          <IconButton
```

Comment:
I'd say let's keep using the internal PlayButton (it's used in the `TimelineComponent`), but refactor it to use iTwinUI buttons/icons.

Reply:
If the `<PlayButton>` swaps between 2 different buttons (play or pause), I think that might be an accessibility concern.  Can we use 1 button and swap the action + icon?

Reply:
Yes, we definitely can (just pass different props to `IconButton`).